tools/ci/appveyor/appveyor_build.sh

```diff
@@ -95,9 +95,15 @@ function set_configuration {
 
 PARALLEL_URL='https://git.savannah.gnu.org/cgit/parallel.git/plain/src/parallel'
 APPVEYOR_BUILD_FOLDER=$(echo "$APPVEYOR_BUILD_FOLDER" | cygpath -f -)
+FULL_BUILD_PREFIX="$APPVEYOR_BUILD_FOLDER/../$BUILD_PREFIX"
 FLEXDLLROOT="$PROGRAMFILES/flexdll"
 OCAMLROOT=$(echo "$OCAMLROOT" | cygpath -f - -m)
 
+# reset and set the git clone as safe
+git config --global --add safe.directory ''
```

Comment:
It's a publicly hosted CI system - I think adding `*` here simply disables this "feature"!